sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 - Removed `GetDfsUri` in all clients since they are currently implementation details.
 - `DataLakePathClient` can no longer set permissions with `SetAccessControl`, instead, a new API `SetPermissions` is created for such functionality. Renamed the original API to `SetAccessControlList` to be more precise.
+- `ContentRange` in `PathGetPropertiesResult` and `ReadDataLakeFileResult` are now `Azure::Core::Http::Range`.
```

Comment:
Why do we need `ContentRange` in `PathGetPropertiesResult`?

Reply:
It is returned via REST. However, this result is not used anywhere in the convenience layer.

Reply:
Is it exposed to end user? If so, I have concern about this. We should discuss how to deal with it

Reply:
It is not going to be exposed to the user shortly, after I have time to make every returned result in the protocol layer in `details` namespace.